src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,69 @@
+package duke;
+
+import java.time.LocalDate;
+import java.util.Scanner;
+
+public class Parser {
+    public static void parse(String input, Scanner sc, TaskList tasks, Storage storage) {
+        while (!input.equalsIgnoreCase("bye")) {
+            Ui.showLine();
+            int id;
+            switch(input) {
+                case "list" :
```

Comment:
Shouldn't the indentation for the cases in your switch block be aligned with the switch statement? 🤔